app/src/main/AndroidManifest.xml

```diff
@@ -50,6 +50,18 @@
       android:name="com.google.mlkit.vision.DEPENDENCIES"
       android:value="barcode" />
 
+    <meta-data
+      android:name="io.sentry.dsn"
+      android:value="${sentryDsn}" />
+
+    <meta-data
+      android:name="io.sentry.environment"
+      android:value="${sentryEnvironment}" />
```

Comment:
Huh, I didn't know this was possible.

Reply:
Yep, we can do this using `manifestPlaceHolders`: https://developer.android.com/studio/build/manifest-build-variables

Reply:
@msasikanth can you explain me why it was better to add these in the manifest than the `build.gradle` file?

Reply:
WDYM? Sentry initialization after the update will be done automatically from the data provided in the manifest. PTAL at this: https://docs.sentry.io/platforms/android/